docs/how_to/cost_analysis.md

```diff
@@ -0,0 +1,31 @@
+# Cost Analysis
+
```

Comment:
(for later) to call out cost of embeddings? I remember you mentioned those were expensive too

Reply:
Yeah  the embedding section could definitely be a bit more fleshed out